docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -31,6 +31,7 @@ If you have a management cluster running EKS Anywhere version v0.15, you can suc
 When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only apply your changes in the workload cluster spec, but also any new improvements included in the new EKS Anywhere controller that was upgraded on the management cluster.
 The changes in the EKS Anywhere controller can trigger a machine rollout on the workload cluster during upgrade, even if the changes to the workload cluster spec didn't require one (for example, scaling down a worker node group).
 
+To upgrade a cluster to v0.18.x `eksctl anywhere` version, it is madatory to upgrade the cluster first to `eksctl anywhere` version v0.17.x. From v0.18.0 onwards, CLI waits for 1 hour combined for waiting for etcd to be ready, control plane to be ready, default CNI to be configured, worker nodes to be ready and cluster to be ready state. It is highly recommended to run the `upgrade` command with the `--no-timeouts` flag to override 1 hour timeout behavior. The `osImageURL` name for the cluster should have a cluster kubernetes version or workerNodeGroupConfiguration.kubernetesVersion version(in case of modular upgrade) in the name. If the kubernetes version is 1.24, the `osImageURL` name should include 1.24, 1_24, 1-24 or 124.
```

Comment:
- I wonder if this info should be spread in multiple paragraphs or even bullet points. in its current form it seems easy to miss and there is no much cohesion between the sentences.
- why is no-timeouts highly recommended? We should say that now it's the only option to tweak the timeouts, but why do we recommend it?

Reply:
- Agree, done.
- I have tweaked the sentence. Pl let me know if you want to change it.